File: Examples/example/marketdatafeed/TicksFalloffStatement.cs
using System;

using net.esper.client;

namespace net.esper.example.marketdatafeed
{
    public class TicksFalloffStatement
    {
        private EPStatement statement;

        public TicksFalloffStatement(EPAdministrator admin)
        {
            String stmt = "select Feed, avg(cnt) as avgCnt, cnt as feedCnt from TicksPerSecond.win:time(10 sec) " +
                          "group by Feed " +
                          "having cnt < avg(cnt) * 0.75 ";

            statement = admin.CreateEQL(stmt);
        }

        public void AddListener(UpdateListener listener)
        {
            statement.AddListener(listener);
        }
    }
}

File: trunk/examples/src/main/java/net/esper/example/marketdatafeed/TicksFalloffStatement.java
package net.esper.example.marketdatafeed;

import net.esper.client.EPStatement;
import net.esper.client.EPAdministrator;
import net.esper.client.UpdateListener;

public class TicksFalloffStatement
{
    private EPStatement statement;

    public TicksFalloffStatement(EPAdministrator admin)
    {
        String stmt = "select feed, avg(cnt) as avgCnt, cnt as feedCnt from TicksPerSecond.win:time(10 sec) " +
                      "group by feed " +
                      "having cnt < avg(cnt) * 0.75 ";

        statement = admin.createEQL(stmt);
    }

    public void addListener(UpdateListener listener)
    {
        statement.addListener(listener);
    }
}
